api/swagger.yaml

```diff
@@ -173,6 +173,17 @@ tags:
     x-displayName: "System"
 
 definitions:
+  Capabilities:
+    description: "Represents engine capabilities"
+    type: "object"
+    required: ["registry-client-auth"]
```

Comment:
I don't think it makes sense to declare any of the capability-flag properties as required. Any capability flags added after the first release would have to be optional as older daemons wouldn't be marshaling them. So it strikes me as odd to special-case the capabilities added in the initial implementation.

By declaring all capability properties as optional, we would be afforded more expressive power with the response, if we chose to make use of it. We could define that the presence of a capability with a false value signals that the daemon knows about the capability, despite it being disabled. As a corollary, the absence of a capability would therefore imply that the daemon is unaware of the capability at all.